README.md

```diff
@@ -73,10 +73,10 @@ An object is identified as a built-in using:
 
 #### `[[Builtin]]` internal slot
 
-All built-ins, with the exception of the intrinsic object `%ObjectPrototype%`,
-would have a `[[Builtin]]` internal slot with a string value identifying the
-name of the built-in. For instance, for the `%Math%` intrinsic object, the
-value of `[[Builtin]]` is `'Math'`.
+All built-ins, with the exception of the intrinsic objects `%ObjectPrototype%`,
+and `%Object%` have a `[[Builtin]]` internal slot with a string value
```

Comment:
Perhaps can be simply say that unlike the other intrinsic objects, whose `[[Builtin]]` is equal to their constructor name (e.g. `%Date%.[[Builtin]] == `Date`), `%Object%.[[Builtin]]` would equal `'object'`, so that the `Builtin.typeof` result is the same and `Builtin.is(Object)` would be `true`?